gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
 #Wed Feb 12 10:23:21 PST 2020
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.1-all.zip
+#distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.2-all.zip
+distributionUrl=https\://mytest.artifactory.com/artifactory/gradle-dist/gradle-7.4.2-all.zip
```

Comment:
When you open the project for the first time, you'll get an error: 
`Unknown host 'mytest.artifactory.com'. You may need to adjust the proxy settings in Gradle.
Enable Gradle 'offline mode' and sync project
Learn about configuring HTTP proxies in Gradle`